+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Fri Sep  1 17:32:44 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL setroubleshoot-server
+ pkgs=setroubleshoot-server
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs setroubleshoot-server --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpe6de9i6n
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpe6de9i6n
+ systemctl start firewalld
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Fri Sep  1 17:33:15 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase.'
+ message='Start executing testcase.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase.'
Fri Sep  1 17:33:16 2023 - INFO  - Start executing testcase.
+ grep '^PasswordAuthentication yes' /etc/ssh/sshd_config
PasswordAuthentication yes
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ semanage port --delete -t ssh_port_t -p tcp 36
libsemanage.semanage_read_policydb: Could not open kernel policy /var/lib/selinux/targeted/active/policy.kern for reading. (No such file or directory).
FileNotFoundError: No such file or directory
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_sec_ssh.sh line 33'
+ message='oe_test_sec_ssh.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_sec_ssh.sh line 33'
Fri Sep  1 17:33:19 2023 - ERROR - oe_test_sec_ssh.sh line 33
+ return 0
+ semanage port -a -t ssh_port_t -p tcp 36
libsemanage.semanage_read_policydb: Could not open kernel policy /var/lib/selinux/targeted/active/policy.kern for reading. (No such file or directory).
FileNotFoundError: No such file or directory
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_sec_ssh.sh line 35'
+ message='oe_test_sec_ssh.sh line 35'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_sec_ssh.sh line 35'
Fri Sep  1 17:33:22 2023 - ERROR - oe_test_sec_ssh.sh line 35
+ return 0
+ firewall-cmd --remove-port 36/tcp
ERROR:dbus.proxies:Introspect error on :1.336:/org/fedoraproject/FirewallD1: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NoReply: Message recipient disconnected from message bus without replying
Error: The name :1.336 was not provided by any .service files
+ firewall-cmd --add-port 36/tcp
FirewallD is not running
+ CHECK_RESULT 252
+ actual_result=252
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 252 ']'
+ '[' 0 -eq 0 ']'
+ test 252x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_sec_ssh.sh line 38'
+ message='oe_test_sec_ssh.sh line 38'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_sec_ssh.sh line 38'
Fri Sep  1 17:33:32 2023 - ERROR - oe_test_sec_ssh.sh line 38
+ return 0
+ firewall-cmd --runtime-to-permanent
FirewallD is not running
+ CHECK_RESULT 252
+ actual_result=252
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 252 ']'
+ '[' 0 -eq 0 ']'
+ test 252x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_sec_ssh.sh line 40'
+ message='oe_test_sec_ssh.sh line 40'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_sec_ssh.sh line 40'
Fri Sep  1 17:33:35 2023 - ERROR - oe_test_sec_ssh.sh line 40
+ return 0
+ LOG_INFO 'Finish testcase execution.'
+ message='Finish testcase execution.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish testcase execution.'
Fri Sep  1 17:33:35 2023 - INFO  - Finish testcase execution.
+ CASE_RESULT 0
+ case_re=0
+ test -z 4
+ test 4 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Sep  1 17:33:35 2023 - ERROR - failed to execute the case.
+ exit 4
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Sep  1 17:33:36 2023 - INFO  - start environment cleanup.
+ firewall-cmd --remove-port 36/tcp
FirewallD is not running
+ firewall-cmd --runtime-to-permanent
FirewallD is not running
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpe6de9i6n ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpe6de9i6n
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Fri Sep  1 17:33:55 2023 - INFO  - Finish environment cleanup!
